packages/p/p5.js.json

```
@@ -27,6 +27,12 @@
         "files": [
           "**/p5*"
         ]
+      },
+      {
+        "basePath": "",
+        "files": [
+          "translations/**"
```

Open globs are dangerous, please could we make this more specific?
From the PR it sounds like this glob would pick up everything?
```suggestion
          "translations/*/*.json"
```

---

Thanks! I think this should work. Made the change.